pkg/controller/common_test.go

```diff
@@ -475,6 +479,21 @@ func initState(client client.Client, azClient azdisk.Interface, kubeClient kuber
 	config := &azdiskv1beta2.AzDiskDriverConfiguration{
 		DriverName:      consts.DefaultDriverName,
 		ObjectNamespace: consts.DefaultAzureDiskCrdNamespace,
+		EventTTLInSec:   testEventTTLInSec,
+	}
+
+	vflag := flag.Lookup("v")
+	if vflag == nil {
+		overwriteVerbosity := true
+		klog.InitFlags(nil)
+		flag.Visit(func(f *flag.Flag) {
+			if f.Name == "v" {
+				overwriteVerbosity = false // use the explicitly provided verbosity
+			}
+		})
+		if overwriteVerbosity {
+			flag.Set("v", strconv.Itoa(defaultTestVerbosity)) // use `defaultTestVerbosity` instead of the default verbosity of 0
```

Comment:
We shouldn't overwrite the verbosity if not specified.

Reply:
for our unit tests, you don't think the default verbosity of 0 is too low? the default verbosity as it is now seems geared towards production; I noticed other k8s projects using non-static klogging w/ 5 as their default verbosity for testing, but we use a mix of traditional klog calls (reporting mostly non-zero level events) & workflow logging (reporting mostly zero-level events) & useful messages while debugging failed unit tests might be missed

Reply:
The unit tests already produce a lot of output (when using the `go test -v` option) which makes it difficult to find the failing test case. Providing an option to increase verbosity to help debug a specific failing case when one manually would be preferable to emitting additional logs by default.

Reply:
We could only log the output of tests which succeed by removing the go test `-v` option: https://pkg.go.dev/cmd/go#hdr-Testing_flags; even with the go test `-v` option, the default klog verbosity is 0, but there are non-zero messages which would be useful for our workflows & makes...
Nevertheless, I'll remove this logic, if we still want to keep everything as is, but I'd definitely agree that our unit tests produce a _lot_ of output esp. due to our workflow logging.

Reply:
As we discussed, I am fine with adding an option to enable additional logging when the unit tests are run manually, but it should not emit them by default.